wizard/views/metadata.py

```diff
@@ -22,6 +23,8 @@
 
 
 def render_metadata():
+    if st.session_state[CurrentStep] == "start":
+        "Start Entering Metadata about your files:"
```

Comment:
Do you need a `st.write` or can Streamlit read this string?

Reply:
removed anyways, was cruft.